### PR TITLE
Update e2e tests to confirm PropagatedClaims are propagated

### PR DESCRIPTION
**Pull request by sbryzak**
Update e2e tests to pass with new API changes

### CODE REVIEW

**Comment by mfrancisc** [comment]
/retest

infra

File: testsupport/user_assertions.go [hunk 1]
```diff
@@ -219,11 +223,18 @@ func VerifySpaceRelatedResources(t *testing.T, awaitilities wait.Awaitilities, u
 	tiers.VerifyNSTemplateSet(t, hostAwait, memberAwait, nsTemplateSet, tierChecks)
 }
 
-func ExpectedUserAccount(userID string, originalSub string) toolchainv1alpha1.UserAccountSpec {
+func ExpectedUserAccount(sub, userID, accountID, originalSub, email string) toolchainv1alpha1.UserAccountSpec {
```
Comment (alexeykazakov) [inline testsupport/user_assertions.go:226]: I wonder if this function could use PropogatedClaims param instead?

```suggestion
func ExpectedUserAccount(claims *PropogatedClaims) toolchainv1alpha1.UserAccountSpec {
```

WDYT?

Comment (sbryzak) [reply testsupport/user_assertions.go]: Good idea, implemented.

File: testsupport/user_assertions.go [hunk 2]
```diff
@@ -219,11 +219,18 @@ func VerifySpaceRelatedResources(t *testing.T, awaitilities wait.Awaitilities, u
...
 		Disabled:    false,
-		OriginalSub: originalSub,
+		OriginalSub: claims.OriginalSub,
+		PropagatedClaims: toolchainv1alpha1.PropagatedClaims{
+			UserID:      claims.UserID,
+			AccountID:   claims.AccountID,
+			OriginalSub: claims.OriginalSub,
+			Sub:         claims.Sub,
+			Email:       claims.Email,
+		},
```
Comment (alexeykazakov) [inline testsupport/user_assertions.go:233]:
```suggestion
		PropagatedClaims: claims,
```
no?

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**